### PR TITLE
Update help text for nodes

### PR DESCRIPTION
**Pull request by knolleary**
Updates help text for the nodes.

It also updates the node implementation to refine the message properties it uses.

The `Project In` node sets `msg.projectLink`. This contains information about where the message came from.

```
{ "projectId": "e9f0dc64-c5b5-40b9-b663-b0576a8a9f34", "topic": "command" }
```

The `Project Call` node includes a `callStack` property under this:

image

### CODE REVIEW

File: nodes/project-link.html [hunk 1]
```diff
@@ -368,94 +368,87 @@
...
         <dd>
-            The <code>msg</code> object sent from a Project Link Out node set to
-            <code>return</code> mode
+            This property contains information about the source of the message.
+            <ul>
+                <li><code>topic</code> - the topic the message was received on</li>
+            </ul>
         </dd>
     </dl>
 </script>
```
Comment (Steve-Mcl) [inline nodes/project-link.html:454]:
```suggestion

</script>
```

Add new line for linting requirements

**Review by Steve-Mcl — COMMENTED** [review]
Other than lint issue - OK 👍

File: nodes/project-link.html (hunk 1 above)
Comment (knolleary) [reply nodes/project-link.html:454]: I suspect you mean a newline *after* the `</script>` not before

**Review by Steve-Mcl — APPROVED** [review]

**Comment by knolleary** [comment]
This is now ready for full review - it updates the README, node help and the message properties used by the nodes.

File: nodes/project-link.js [hunk 2]
```diff
@@ -746,11 +741,12 @@ module.exports = function (RED) {
         node.project = n.project
         node.subTopic = n.topic
         node.topic = buildLinkTopic(node, node.project, node.subTopic, false, false)
-        node.responseTopic = buildLinkTopic(node, node.project, node.subTopic, false, true)
+        node.responseTopic = buildLinkTopic(node, RED.settings.flowforge.projectID, node.subTopic, false, true)
         let timeout = parseFloat(n.timeout || '30') * 1000
         if (isNaN(timeout)) {
             timeout = 30000
         }
+        const messageEvents = {}
```
Comment (Steve-Mcl) [inline nodes/project-link.js:749]:
```suggestion
        /** @type {MessageEvents} */
        const messageEvents = {}
```

Restore type hint - better dev experience with autocompletion and property listing of object and properties` properties.

**Review by Steve-Mcl — APPROVED** [review]
all goodness. local tests and spellcheck done

**Pull request merged**